static/css/restyle/restyle.less

```diff
@@ -1996,6 +1992,15 @@ body.editor-tools {
       .queue-inner {
         border-radius: 0;
         border: none;
+
+        .data-grid th.ordered {
+          background: none;
+          background-color: #ffffff;
```

Comment:
Can you set `background: #fff` instead of setting it to `none` then setting `background-color` specifically?